internal/xds/bootstrap/bootstrap.yaml.tpl

```diff
@@ -95,30 +95,24 @@ static_resources:
                       exact: GET
                 route:
                   cluster: prometheus_stats
-                {{- if .EnablePrometheusCompression }}
-                typed_per_filter_config:
-                  envoy.filters.http.compression:
-                    "@type": type.googleapis.com/envoy.extensions.filters.http.compressor.v3.CompressorPerRoute
-                    {{- if eq .PrometheusCompressionLibrary "gzip"}}
-                    compressor_library:
-                      name: text_optimized
-                      typed_config:
-                        "@type": type.googleapis.com/envoy.extensions.compression.gzip.compressor.v3.Gzip
-                    {{- end }}
-                    {{- if eq .PrometheusCompressionLibrary "brotli"}}
-                    compressor_library:
-                      name: text_optimized
-                      typed_config:
-                        "@type": type.googleapis.com/envoy.extensions.compression.brotli.compressor.v3.Brotli
-                    {{- end }}
-                    {{- if eq .PrometheusCompressionLibrary "zstd"}}
-                    compressor_library:
-                      name: text_optimized
-                      typed_config:
-                        "@type": type.googleapis.com/envoy.extensions.compression.zstd.compressor.v3.Zstd
-                    {{- end }}
-                {{- end }}
           http_filters:
+          {{- if .EnablePrometheusCompression }}
```

Comment:
shouldnt this be scoped only to the prom route ?

Reply:
we didn't share the listener now

Reply:
is https://github.com/envoyproxy/gateway/pull/5363/commits/5e18fb6938985e7450bc18ae7c3f5f6bbcdb6e91 what you want?

Reply:
ah its a unique listener so this is safe, thanks